packages/transaction-controller/src/helpers/AccountsApiRemoteTransactionSource.ts

```diff
@@ -80,27 +80,15 @@ export class AccountsApiRemoteTransactionSource
   async #getTransactions(request: RemoteTransactionSourceRequest) {
     log('Getting transactions', request);
 
-    const { address, cache, chainIds: requestedChainIds } = request;
+    const { address, cache } = request;
 
-    const chainIds = requestedChainIds.filter((chainId) =>
-      SUPPORTED_CHAIN_IDS.includes(chainId),
-    );
-
-    const unsupportedChainIds = requestedChainIds.filter(
-      (chainId) => !chainIds.includes(chainId),
-    );
-
-    if (unsupportedChainIds.length) {
-      log('Ignoring unsupported chain IDs', unsupportedChainIds);
-    }
-
-    const cursor = this.#getCacheCursor(cache, chainIds, address);
+    const cursor = this.#getCacheCursor(cache, SUPPORTED_CHAIN_IDS, address);
```

Comment:
For audit sake, I agree that using a constant is safer to begin with so we don't suddenly start polling more chains.